### PR TITLE
Delete some obsolete documentation and configuration files

### PR DESCRIPTION
**Pull request by thilo-m**

### CODE REVIEW

**Review by abhijeetnarvekar — COMMENTED** [review]
What is the future plan after deleting? Is the plan to add new documentation for coding guidance. Because the name of the repository doesn't make sense after deleting most of  topics related to coding guidance.

File: topics/performance.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# Performance
-
-During development, the focus should be on readability, not on locally optimized performance.
-It's easier to optimize readable code than to make code with good performance readable.
-If the code is unreadable and not performant, what do you do?
-
-* Nevertheless, user actions should complete within 200 ms. If they're slower, try to optimize your code.
```
Comment (abhijeetnarvekar) [inline topics/performance.md]: The 200ms is important to external developers and is still relevant. Can we add it to some other section?

File: topics/logging.md [hunk 2]
```diff
@@ -1,146 +0,0 @@
...
-* IP addresses, port numbers, process IDs
-* Context of a finding
-* Surfaces of terms
-* Passwords
-* Authentication tokens or session IDs if they act as a hard-to-guess, shared secret
-* File system paths
-* SQL queries with values
-* To a certain degree, also stack traces since they reveal how things work.
-
-Use the following guide to decide what information to log at which level.
```
Comment (abhijeetnarvekar) [inline topics/logging.md]: This section is important information for external devs. Especially logging text surfaces from user content, devs might accidentally log data from content in error messages

File: topics/performance.md (hunk 1 above)
Comment (thilo-m) [reply topics/performance.md]: I restored the whole file again.

File: topics/logging.md (hunk 2 above)
Comment (thilo-m) [reply topics/logging.md]: I restored the whole file again.

**Review by abhijeetnarvekar — APPROVED** [review]

**Pull request merged**